eu.cichy1173.tabela.yml

```diff
@@ -0,0 +1,32 @@
+app-id: eu.cichy1173.tabela
+runtime: org.kde.Platform
+runtime-version: '6.2'
+sdk: org.kde.Sdk
+command: tabela
+build-options:
+ build-args:
+  - --share=network
```

Comment:
this is not allowed on our builders. build must be done offline.

Reply:
Hi. Thank You for your response. Can you tell me how to fix this? I am sorry, but I am new in this. 

I built this locally from dir and using codeberg repository, but for building fro repo, this build-arg was needed.